devito/function.py

```diff
@@ -436,8 +437,7 @@ def __init__(self, *args, **kwargs):
             else:
                 raise ValueError("'padding' must be int or %d-tuple of ints" % self.ndim)
             self._padding = padding
-
-            # Dynamically add derivative short-cuts
+            # Initialize derivatives
```

Comment:
nitpicking: leave blank line after ``self._padding = ...``

Of course you can do this in one of the other open PRs